plugin/src/test/groovy/com/novoda/buildproperties/EnvironmentPropertiesEntriesTest.groovy

```diff
@@ -1,70 +0,0 @@
-package com.novoda.buildproperties
-
-import com.novoda.buildproperties.test.EntrySubject
-import org.gradle.api.Project
-import org.gradle.testfixtures.ProjectBuilder
-import org.junit.Before
-import org.junit.Rule
-import org.junit.Test
-import org.junit.contrib.java.lang.system.EnvironmentVariables
-import org.junit.rules.TemporaryFolder
-
-import static com.google.common.truth.Truth.assertThat
-
-class EnvironmentPropertiesEntriesTest {
```

Comment:
Just to understand. I guess the integration test still tests, setting the real EnvironmentVariable thanks to the change in the sample? 💯